src/main/java/frc/team2412/robot/subsystems/LauncherSubsystem.java

```diff
@@ -134,8 +144,11 @@ public void configMotors() {
 		// PID
 		launcherAngleOnePIDController.setP(5.0);
 		launcherAngleOnePIDController.setI(0);
-		launcherAngleOnePIDController.setD(0);
-		launcherAngleOnePIDController.setFF(0);
+		launcherAngleOnePIDController.setD(3);
+		launcherAngleOnePIDController.setFF(
```

Comment:
FF should be recalculated every time the angle of the pivot is set. You can include FF as a parameter when you set the motor reference to accomplish that

Reply:
Okay, so would i just run everything in .setFF in .setReference?

Reply:
Pretty much. See https://github.com/robototes/Crescendo2024/pull/92/files#diff-d8d7d840be58e4901f0702d7c23a5a27cf7cbb95f4b21d2ce2fa8e4ab36a76e9R189 for an example